atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -535,7 +535,7 @@ void createTable(@QueryParam("tableRef") TableReference tableRef, byte[] tableMe
     @Path("put-metadata-for-tables")
     @Consumes(MediaType.APPLICATION_JSON)
     @Idempotent
-    void putMetadataForTables(final Map<TableReference, byte[]> tableRefToMetadata);
+    void putMetadataForTables(Map<TableReference, byte[]> tableRefToMetadata);
```

Comment:
This was an annoying warning that triggered OCE in IDE thing. `KeyValueService` is an interface and this is not a default implementation, so there is no need to mark a parameter as final.